packages/fxa-payments-server/src/routes/Product/PlanDetails/index.test.tsx

```diff
@@ -6,18 +6,16 @@ import { PlanDetails } from './index';
 afterEach(cleanup);
 
 it('renders default details for unknown product ID', async () => {
-  const { findByText, queryByTestId } = render(<Subject />);
-  await findByText('Loaded');
-  expect(queryByTestId('plan-default')).toBeInTheDocument();
+  const { findByTestId } = render(<Subject />);
+  await findByTestId('plan-default');
```

Comment:
These changes seem kinda unrelated, do you want these in this branch?

Reply:
There seemed to be a race condition in the test that got exacerbated by the React upgrade. These tests started failing until I reworked what they were async waiting for